### ISSUE TITLE
Error line number doesn't show anymore in debug console when executing simple JavaScript program

### ISSUE THREAD

**Issue by jeremymouzin** [microsoft/vscode#141142]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.63.2
- OS Version: MacOS Monterey 12.1
- Node version: 17.2.0

Steps to Reproduce:

1. Create a test.js file with following content:
```js
let variable = null;
console.log(variable.something);
```
2. Run this code with CTRL+F5, select Node.js environment to run it
3. As expected, in the debug console it shows the following error "Uncaught TypeError: Cannot read properties of null (reading 'prop')" **but there is no mention of the error line number as in previous VSCode versions**.

Screenshot of where the error line number should appear in debug console:
![error vscode](https://user-images.githubusercontent.com/20302631/150518542-5040ed64-7311-47f3-b095-2e62c06ba09f.jpeg)

**Comment by weinand** [comment]
@jeremymouzin No, VS Code did not show the line number of errors on the right hand side.
Here is a screenshot from VS Code 1.60.2 (September 2021):

<img width="756" alt="CleanShot 2022-01-24 at 11 47 34@2x" src="https://user-images.githubusercontent.com/1898161/150769219-6312d36a-3ab2-4f76-81fb-0dd551373e2f.png">

You are referring to the source line information of `console.log` statements:
<img width="767" alt="CleanShot 2022-01-24 at 11 44 19@2x" src="https://user-images.githubusercontent.com/1898161/150769635-1967d955-3dd7-47ed-bb4d-6a7968a8358f.png">

In your case the `console.log` isn't executed because of the TypeError - so there is no source line shown.

---

@connor4312 we could probably extract the source line from the error's stack?

**Comment by jeremymouzin** [comment]
@weinand Thanks for your feedback.

I was not referring to the source line information of the console.log statement but indeed I did misplaced the green square on my screenshot. It should be at the end of the line on the left as in this case:

![CleanShot 2022-01-24 at 12 16 51](https://user-images.githubusercontent.com/20302631/150773839-0d1f62a0-aede-4e99-9848-97eac1c2132c.jpg)

With the code above the error line information is shown and the console.log statement is still not executed.
It's quite confusing to have some errors with the error line info and some without.

Also with outputCapture set on "std" I always get the correct error line info:

![outputCapture std](https://user-images.githubusercontent.com/20302631/150774231-39eb9cab-b875-49d4-927f-6e9eccfa4aaf.jpg)

Always having the error line info no matter the error would be less confusing and of a great value for debugging long scripts.

Thanks a lot for your time and I'm sorry if I added a useless bug report, maybe it should be converted to a feature request?
